src/index.js

```diff
@@ -6,7 +6,7 @@ import * as serviceWorker from './serviceWorker';
 import { BrowserRouter } from 'react-router-dom'
 import axios from 'axios'
 
-axios.defaults.baseURL = 'https://glocal-news.herokuapp.com'
+axios.defaults.baseURL = 'http://localhost:3002/'
```

Comment:
Shouldn't this be changed "back" to the deployed site from Heroku, before being merged?

Reply:
yes, will do!